staging/src/k8s.io/kubectl/pkg/cmd/drain/drain.go

```diff
@@ -145,9 +145,10 @@ func NewDrainCmdOptions(f cmdutil.Factory, ioStreams genericclioptions.IOStreams
 		PrintFlags: genericclioptions.NewPrintFlags("drained").WithTypeSetter(scheme.Scheme),
 		IOStreams:  ioStreams,
 		drainer: &drain.Helper{
-			GracePeriodSeconds: -1,
-			Out:                ioStreams.Out,
-			ErrOut:             ioStreams.ErrOut,
+			GracePeriodSeconds:  -1,
```

Comment:
I would not change the spacing/order for GracePeriodSeconds, Out, ErrOut to avoid show-up in the diff.